core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@vahidhashemian Haha, I'm not sure I see the source of the confusion. The MM processes are basically just a consumer group. They all subscribe to the same set of topics and they depend on the assignment protocol to balance the partitions from those topics among the group. So this acquires that they all have a compatible assignment process, right?